docs/administration/general-settings/_index.md

```diff
@@ -27,3 +27,9 @@ denied: The system is in read only mode. Any modification is prohibited.
 Use the **Project Creation** drop-down menu to set which users can create projects. Select **Everyone** to allow all users to create projects. Select **Admin Only** to allow only users with the Harbor system administrator role to create projects.
 
 ![browse project](../../img/new-proj-create.png)
+
+## Skip updating the pull time for scanner
```

Comment:
Not sure if skip is the right word here, will reach you offline to get some context

Reply:
I think "Retain image last pull time on scanning" is better